Source/ChocolateyGui/Properties/Resources.sv-SE.resx

```diff
@@ -0,0 +1,357 @@
+  </data>
+  <data name="SettingsView_ButtonNew" xml:space="preserve">
+    <value>Ny</value>
+  </data>
+  <data name="SettingsView_ButtonRemove" xml:space="preserve">
+    <value>Ta bort</value>
+  </data>
+  <data name="SettingsView_ButtonSave" xml:space="preserve">
+    <value>Spara</value>
+  </data>
+  <data name="SettingsView_ChocoGui" xml:space="preserve">
+    <value>Chocolatey Gui</value>
+  </data>
+  <data name="SettingsView_Credits" xml:space="preserve">
+    <value>Tack till</value>
+  </data>
+  <data name="SettingsView_Features" xml:space="preserve">
+    <value>Funktioner</value>
+  </data>
+  <data name="SettingsView_Options" xml:space="preserve">
+    <value>Alternativ</value>
+  </data>
+  <data name="SettingsView_PropertyDescription" xml:space="preserve">
+    <value>Beskrivning</value>
+  </data>
+  <data name="SettingsView_PropertyName" xml:space="preserve">
+    <value>Namn</value>
+  </data>
+  <data name="SettingsView_PropertyValue" xml:space="preserve">
+    <value>Värde</value>
+  </data>
+  <data name="SettingsView_ReleaseNotes" xml:space="preserve">
+    <value>Ändringslogg</value>
+  </data>
+  <data name="SettingsView_Settings" xml:space="preserve">
+    <value>Inställningar</value>
+  </data>
+  <data name="SettingsView_Sources" xml:space="preserve">
+    <value>Källor</value>
+  </data>
+  <data name="SettingsView_SourcesCertificate" xml:space="preserve">
+    <value>Certifikat</value>
+  </data>
+  <data name="SettingsView_SourcesCertificatePass" xml:space="preserve">
+    <value>Certifikatlösenord</value>
+  </data>
+  <data name="SettingsView_SourcesId" xml:space="preserve">
+    <value>Id</value>
+  </data>
+  <data name="SettingsView_SourcesIsDisabled" xml:space="preserve">
+    <value>Är inaktiverad</value>
+  </data>
+  <data name="SettingsView_SourcesPassword" xml:space="preserve">
+    <value>Lösenord</value>
+  </data>
+  <data name="SettingsView_SourcesPath" xml:space="preserve">
+    <value>Sökväg</value>
+  </data>
+  <data name="SettingsView_SourcesPriority" xml:space="preserve">
+    <value>Prioritet</value>
+  </data>
+  <data name="SettingsView_SourcesSource" xml:space="preserve">
+    <value>Källa</value>
+  </data>
+  <data name="SettingsView_SourcesUsername" xml:space="preserve">
+    <value>Användarnamn</value>
+  </data>
+  <data name="SettingsView_SubheaderChoco" xml:space="preserve">
+    <value>Chocolatey</value>
+  </data>
+  <data name="SettingsView_ToggleElevateByDefault" xml:space="preserve">
+    <value>Höj behörigheten automatiskt</value>
+  </data>
+  <data name="SettingsView_ToggleShowConsoleOutput" xml:space="preserve">
+    <value>Visa konsolutskrift</value>
+  </data>
+  <data name="ShellView_ButtonAbout" xml:space="preserve">
+    <value>om</value>
+  </data>
+  <data name="ShellView_ButtonSettings" xml:space="preserve">
+    <value>inställningar</value>
+  </data>
+  <data name="SourcesView_Choco" xml:space="preserve">
+    <value>chocolatey</value>

```

Comment:
@patriksvensson lowercase c here makes sense, and fits with the English version.  This is to do with the overall look and feel of the app.